nmpolicy/internal/capture/capture_test.go

```diff
@@ -192,3 +186,7 @@ func testResolveFailure(t *testing.T) {
 		assert.Error(t, err)
 	})
 }
+
+func captureResolverWithDefaultStubs() capture.Capture {
```

Comment:
`defaultStubCapturedState(expression string)` and `captureResolverWithDefaultStubs()` should be in the same file.
I would move `captureResolverWithDefaultStubs` to `stub_test`.

Reply:
This is the only test suite that instanciate `Capture` so it has to be here.